core/schemas/mcp.go

```diff
@@ -14,6 +14,7 @@ type MCPConfig struct {
 
 // MCPClientConfig defines tool filtering for an MCP client.
 type MCPClientConfig struct {
+	ID               string            `json:"id"`                          // Client ID
```

Comment:
add a comment to add id, because it is used by public methods as well

Reply:
and lets has 2 validations in MCP client configs (inside code)

- ID should not be empty
- no duplicate names